### PR TITLE
Remove polished package

### PR DESCRIPTION
**Pull request by g4rry420**
#### Where should the reviewer start?
- components => StyledRangeInput.js
- themes => base.js, dark.js, 
- Select => stories => theme.js

#### What testing has been done on this PR?
yarn test-update

#### How should this be manually tested?
Using the storybooks

#### Any background context you want to provide?
As mentioned in #3976 , `polished` is not much useful and adds 10kb to bundlesize. The only thing where polished's `rgba` was useful is in `StyledRangeInput.js` but similar logic can still applied through the already written utils function `getRGBA` which is helpful for our case.

#### What are the relevant issues?
#3976 

#### Screenshots (if appropriate)
No

#### Do the grommet docs need to be updated?
No

### CODE REVIEW

File: src/js/components/Select/stories/theme.js [hunk 1]
```diff
@@ -12,7 +10,7 @@ export const theme = {
       selected: 'neutral-3',
       border: '#e0e0e0',
       focus: '#2196F3',
-      gray: rgba(0, 0, 0, 0.54),
+      gray: 'rgba(0, 0, 0, 0.54)',
```
Comment (taysea) [inline src/js/components/Select/stories/theme.js:13]: confirming the strings around this is intended?

File: src/js/themes/base.js [hunk 2]
```diff
@@ -50,7 +48,7 @@ const lightColors = [
 const focusColor = accentColors[0];
 
 const colors = {
-  active: rgba(221, 221, 221, 0.5),
+  active: 'rgba(221, 221, 221, 0.5)',
```
Comment (taysea) [inline src/js/themes/base.js:51]: Same as above, did the previous rgba without string around still resolve to a string value?

**Review by taysea — COMMENTED** [review]
Thanks for looking into this, just left a couple questions

File: src/js/components/Select/stories/theme.js (hunk 1 above)
Comment (g4rry420) [reply src/js/components/Select/stories/theme.js:13]: Yes, they are. Here, we are using polished's `rgba`.  But, we can use `rgba` provided by css.

**Review by taysea — APPROVED** [review]
I believe this looks good. Snapshot changes are just regenerated class names by styled components but no values seem to have shifted, which is fine.

**Review by ericsoderberghp — APPROVED** [review]

**Comment by ericsoderberghp** [comment]
Thanks for your contribution!

**Pull request merged**